### PR TITLE
Fix cleaning DB with artemis NG

### PR DESCRIPTION
**Pull request by benoit-bst**
With this PR, we can use ARTEMIS NG with the `clean_kirin_db function`

Now if you want to use Artemis NG, you'll do
```
# example to run only guiclet_unique with the new env var
#
# be careful, the host change at each time. To catch the KIRIN IP database do,
# docker inspect navitia-docker-compose_kirin_database_1_f4e6a643f70c
# resp :
# ...
#"IPAddress": "172.18.0.5",
#...

ARTEMIS_KIRIN_DB='dbname=kirin user=navitia host=172.18.0.5 password=navitia' pytest -k guichet_unique_test.py
```
I tested on the old Artemis Jenkins engine and it passed :smile:

### CODE REVIEW

File: artemis/default_settings.py [hunk 1]
```diff
@@ -24,7 +26,7 @@
 
 JORMUNGANDR_DB = 'dbname=jormungandr user=jormungandr host=localhost password=jormungandr'
 
-KIRIN_DB = 'dbname=kirin user=kirin host=localhost password=kirin'
+KIRIN_DB = os.getenv('ARTEMIS_KIRIN_DB', 'dbname=kirin user=kirin host=localhost password=kirin')
```
Comment (pbougue) [inline artemis/default_settings.py]: As we have to change it for other var, not sure this is needed.
Maybe we should just ask people to use a dev_settings (that would be mostly like the default_settings_docker in my PR?)
Anyway depending on what we want I can update my PR accordingly.

Comment (mbounabi) [reply artemis/default_settings.py]: What other var?

Comment (pbougue) [reply artemis/default_settings.py]: When using artemis_NG, as mentionned in the doc, you have to change some VARS in this file, not just the KIRIN_DB one.
That's why I re-used and updated default_settings_docker.py in mine, to ease the changes that have to be done when using artemis_NG locally.

But what Benoit did here doesn't prevent it, it just adds one more way of doing it. So if you want to keep it, it's OK for me.

**Comment by benoit-bst** [comment]
As we said,

- Updated default_settings.py file with getenv for Artemis NG var
- Delete default_settings_docker.py, wich it useless
- Update doc a bit

I launched tests on Artemis NG and it have passsed.

**Comment by kinnou02** [comment]
You can use the following command to get the IP adress without having to think too much:
```
docker inspect --format='{{.NetworkSettings.IPAddress}}' navitia-docker-compose_kirin_database_1_f4e6a643f70c
```

**Pull request merged**